### DISCUSSION TITLE
Scan Cannot Start due to Limited Hardware Resources

### DISCUSSION THREAD

**Discussion by databoy2k** [NightscoutFoundation/xDrip#3487] [Bug reports] [answered]
I'm receiving lots of connection issues lately, often resulting in this status page: 
![Screenshot_20240526-195401](https://github.com/NightscoutFoundation/xDrip/assets/18686442/dc41830f-1d2d-40fa-b055-802cad5b73f1)

G7, Pixel 8, 2024-05-22. Log below:
![Screenshot_20240526-195619](https://github.com/NightscoutFoundation/xDrip/assets/18686442/a34535ba-f3ab-45fc-a1c0-340e5a59ed69)

I'm connected to an ear bud, but disconnecting it doesn't solve the issue.

**Comment by Navid200** [comment]
Do you have "Scan for G5 constantly" enabled?
If yes, why?

If no, what else is running on your Pixel 8?

**Reply by databoy2k** [reply]
No, i do not 

AAPS, tasker, and at the time i had YouTube running in the background. Reboot doesn't seem to help either.

**Reply by Navid200** [reply]
Can you install the previous Nightly, https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.05.15, over your current installation?
Does it make any difference?

**Reply by Navid200** [reply]
Can you post a screenshot of the following page?

Settings > G5/G6/G7/Dex1 Debug Settings

**Reply by databoy2k** [reply]
I did a quick reboot and then installed the last version. No resource complaint but I'm still not getting readings. Log below: 
![Screenshot_20240526-200741](https://github.com/NightscoutFoundation/xDrip/assets/18686442/6ad903e7-fdf3-4572-bb3b-543609c000ec)

**Reply by databoy2k** [reply]
Settings: 
![Screenshot_20240526-201105](https://github.com/NightscoutFoundation/xDrip/assets/18686442/5087d949-025e-4614-a996-b15b6eb0abe1)

**Reply by Navid200** [reply]
Change the last character of the transmitter ID to something else and apply.
Go to the phone Bluetooth settings and forget all starting with DXCM or DX02.
Put all your old used G7 devices away or in the microwave or inside a metal box.
Change the transmitter ID back to the correct value.
Go to the status page and wait for the pair request and approve.

Does it work now?

**Reply by databoy2k** [reply]
I had to throw in a reboot in order to actually get the pairing request, but yes I'm back up now. 

Any troubleshooting that I can do on the newest nightly?

**Reply by Navid200** [reply]
Well, I don't think it is possible without compromising your setup.
I could ask you to enable OB1 debug logging and let the problem happen again and upload so that our lead developer can see the detailed logs.  But, that would require to intentionally mess up your setup.
We don't ask you to compromise your diabetes control for the sake of troubleshooting the code.  That would be a real misunderstanding of priorities on our part.
So, no, let's wait for our lead developer to advise.
Thanks for the offer

@jamorham

**Reply by databoy2k** [reply]
I appreciate it. Tell you what, tag me if that's what's needed to troubleshoot this. I'd rather have a really bad few days than continue to run into this issue, or worse have somebody who really struggles run headlong into this.

**Reply by databoy2k** [reply]
@Navid200 @jamorham I see a few reporting changes in the 2024-05-28 build; are those for this?

**Reply by Navid200** [reply, marked as answer]
I'm sorry I thought I informed you.
Yes, there is a new release to address what you have reported.  I am so sorry.
Please use the following release or any after and tell us if you still experience the problem:
https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.05.28

Thanks so much for reporting this.

**Reply by databoy2k** [reply]
Thanks! Updated - fingers crossed! I'll let you know if this becomes a problem again.

**Reply by Navid200** [reply]
Thanks

**Comment by databoy2k** [comment]
No issues since the patch. I'll close this. Thanks Navid!
